pegasus/src/homepage.rb

```diff
@@ -498,8 +498,7 @@ def self.show_curriculum_banner(request)
   end
 
   def self.show_professional_learning_banner(request)
```

Comment:
Since this banner is shown in more places than just the homepage, I'd remove this method and just add the check `request.locale == "en-US"` to `homepage_below_hero_pd_banner.haml` directly